src/implementations/title-escrow/transferOwner.ts

```diff
@@ -20,22 +20,22 @@ export const endorseChangeOfOwner = async ({
   ...rest
 }: TitleEscrowEndorseChangeOfOwnerCommand): Promise<TransactionReceipt> => {
```

Comment:
In this file and the file name, if its only beneficiary should change to beneficiary, if is for transferring of beneficiary and holder should be Owners right??